docs/whats-new.md

```diff
@@ -22,6 +22,9 @@ Release date: TBD, Target late Aug, 2018
 
 A new experimental module `@deck.gl/json` provides a set of classes that allows deck.gl layers and views to be rendered using JSON specifications.
 
+## stroke, strokeColor
```

Comment:
See my main comment. If we put it in experimental layers let's make clear it is a new layer:

## New Layers

### `OutlineScatterplotLayer` (Experimental) (By @Jianyf-Jianyf)

An enhanced version of the standard deck.gl `ScatterplotLayer` is now available in `@deck.gl/experimental layers. It adds `stroke`, `strokeColor` properties to control outlines.